riscv64-gen.c

```diff
@@ -1250,8 +1250,49 @@ ST_FUNC void gen_opl(int op)
     gen_opil(op, 1);
 }
 
+// TODO: These functions were defined in tccgen.c in mob branch---
```

Comment:
In our branch too. 
Why do you need to add them here @Marie-Joseph ?

Reply:
When adding the configure flag -DONE_SOURCE, these are not needed; I removed them in the latest revision.